images/mediatypes.go

```diff
@@ -10,4 +10,10 @@ const (
 	MediaTypeDockerSchema2Config       = "application/vnd.docker.container.image.v1+json"
 	MediaTypeDockerSchema2Manifest     = "application/vnd.docker.distribution.manifest.v2+json"
 	MediaTypeDockerSchema2ManifestList = "application/vnd.docker.distribution.manifest.list.v2+json"
+	// Checkpoint/Restore Media Types
+	MediaTypeContainerd1Checkpoint        = "application/vnd.containerd.container.checkpoint.tar"
```

Comment:
I think this media type is specific to CRIU implementation, and I guess we can have non-CRIU implementations in the future (for non-runc runtimes, e.g. clear container, runv, and Windows Hyper-V container)

So how about renaming the media type to `application/vnd.containerd.criu....`?